SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/SharedEnumTests.swift

```diff
@@ -33,11 +33,11 @@ class SharedEnumTests: XCTestCase {
     }
 
     func testEnumWithUnnamedData() {
```

Comment:
We can add a new `testEnumWithOpaqueRustData` that tests a named and unnamed variant

```rust
enum TestEnumWithOpaqueRust {
    Named { data: OpaqueRustForEnumTest },
    Unnamed(OpaqueRustForEnumTest)
}
```